docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -88,7 +88,7 @@ You can attach tags to the application manually, instead of enabling automated t
 +
 [NOTE]
 ====
-Analysis engines use standard rules for a comprehensive set of migration targets, but if the target is not included or is a customized framework, custom rules can be added. Custom rules files are validated.
+Analysis engines use standard rules for a comprehensive set of migration targets, but if the target is not included or is a customized framework, custom rules can be added. Only manually uploaded custom rules files are validated.
```

Comment:
@emarcusRH 
Of course, 'rule files' would be grammatically correct, but 'rules files' already appear in many places. Maybe we should change it everywhere, in some future PR for general improvement.
Otherwise, looks fine.

Reply:
I found 2 instances of "rules files", and 3 instances of "rule files" in the repo. I guess it is easy to fix the 2 incorrect instances.